com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -46,7 +47,7 @@ public static void Create(GameObject linkObject, Link.Inertial inertial = null)
 #endif
             if (inertial != null)
             {
-                robotLink.mass = (float)inertial.mass;
+                robotLink.mass = ((float)inertial.mass > 0)?((float)inertial.mass):minMass;
```

Comment:
nit: coding standard to have spaces? e.g. ((float)inertial.mass > 0) ? ((float) inertial.mass) : minMass;